pandas/tests/arrays/string_/test_string.py

```diff
@@ -230,6 +230,26 @@ def test_reduce(skipna):
     assert result == "abc"
 
 
+@pytest.mark.parametrize("method", ["min", "max"])
+@pytest.mark.parametrize("skipna", [True, False])
+def test_min_max(method, skipna):
+    arr = pd.Series(["a", "b", "c", None], dtype="string")
+    result = getattr(arr, method)(skipna=skipna)
+    if skipna:
+        expected = "a" if method == "min" else "c"
+        assert result == expected
+    else:
+        assert result is pd.NA
+
+
+@pytest.mark.parametrize("method", ["min", "max"])
+def test_min_max_numpy(method):
+    arr = pd.Series(["a", "b", "c", None], dtype="string")
+    result = getattr(np, method)(arr)
+    expected = "a" if method == "min" else "c"
```

Comment:
why isn't this return the same, IIRC numpy just dispatches back to the array, so this should yield the same as above?

Reply:
It's pretty much the same test, I only separated it out so we're only testing numpy min / max once (no need to parametrize over skipna). I could add a few lines to the test above and remove this if that's less clutter

Reply:
I think it is fine (cleaner) to keep the tests separate like this